fw/http_frame.c

```diff
@@ -73,19 +73,6 @@ typedef enum {
 	HTTP2_RECV_APP_DATA_POST
```

Comment:
Please update the copyright year of all the updated files and/or initialize the git hook, which does it automatically